src/organisms/__tests__/mock_data/mockOrganismState.js

```diff
@@ -0,0 +1,118 @@
+const mockCategoryList = [
+  {
+    material_ids: [1, 2, 4],
+    family_id: 1,
+    description: "first family",
+    family_type_id: 1
+  },
+  {
+    material_ids: [3, 5, 7],
+    family_id: 2,
+    description: "second family",
+    family_type_id: 2
+  }
+];
+
+const mockMaterials = [
+  {
+    id: 1,
+    material_id: 1,
+    description: "first material",
+    long_description: "the primary",
+    bin_trash: false,
```

Comment:
how come for the trash, recycle, and compost bin we have either have triple false or 2 true and a false? We would want to have 1 be true so the item displays the same way it would for correct disposal determination. I don't believe I see tests for testing that yet, so it isn't a problem but may be one in the future.